instrumentation/aws-sdk/aws-sdk-2.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awssdk/v2_2/AwsSdkInstrumentationModule.java

```diff
@@ -22,17 +22,7 @@ public AwsSdkInstrumentationModule() {
 
   @Override
   public String[] additionalHelperClassNames() {
-    return new String[] {
-      "io.opentelemetry.javaagent.instrumentation.awssdk.v2_2.TracingExecutionInterceptor",
-      "io.opentelemetry.instrumentation.awssdk.v2_2.AwsSdk",
-      "io.opentelemetry.instrumentation.awssdk.v2_2.AwsSdkHttpClientTracer",
-      "io.opentelemetry.instrumentation.awssdk.v2_2.AwsSdkInjectAdapter",
-      "io.opentelemetry.instrumentation.awssdk.v2_2.RequestType",
-      "io.opentelemetry.instrumentation.awssdk.v2_2.SdkRequestDecorator",
-      "io.opentelemetry.instrumentation.awssdk.v2_2.DbRequestDecorator",
-      "io.opentelemetry.instrumentation.awssdk.v2_2.TracingExecutionInterceptor",
-      "io.opentelemetry.extension.trace.propagation.AwsXRayPropagator"
-    };
+    return new String[] {"io.opentelemetry.extension.trace.propagation.AwsXRayPropagator"};
```

Comment:
Thanks a lot for the change!!! Just checking, will muzzle task fail if this class isn't included now?

Reply:
Yes, it will:

```
FAILED MUZZLE VALIDATION: io.opentelemetry.javaagent.instrumentation.awssdk.v2_2.AwsSdkInstrumentationModule mismatches:
-- io.opentelemetry.instrumentation.awssdk.v2_2.AwsSdkHttpClientTracer:39 Missing class io.opentelemetry.extension.trace.propagation.AwsXRayPropagator
```